root/static/scripts/common/constants/locales.json

```diff
@@ -782,12 +956,24 @@
   "vai_Latn_LR":"Vai Liberia Latin",
   "vai_Vaii":"Vai Vai",
   "vai_Vaii_LR":"Vai Liberia Vai",
+  "ve":"Venda",
+  "ve_ZA":"Venda South Africa",
+  "vec":"Venetian",
+  "vec_IT":"Venetian Italy",
   "vi":"Vietnamese",
   "vi_VN":"Vietnamese Vietnam",
+  "vo":"Volapük",
+  "vo_001":"Volapük world",
```

Comment:
And one more. I guess it's because they always want to have *one* sub-locale but it doesn't apply to these or? Generally I'd want to remove them, but I'm not sure if that's sensible with this coming straight from the source.